src/views/WhenWhere.js

```diff
@@ -79,61 +160,63 @@ class WhenWhere extends Component {
 
   render() {
     return (
-      <main>
+      <StyledMain>
         <Helmet>
           <title>Megan and Ringo's Wedding - The Ceremony</title>
           <meta name="description" content={description} />
           <meta content={description} property="og:description" />
           <link rel="canonical" href="https://meganandringo.com/when-where" />
         </Helmet>
-        <StyledArticle left view="0" style={{ backgroundColor: "white" }}>
-          <LabelSection
-            top="10em"
-            left="2.5em"
-            category="Date"
-            title={event.formattedDate.toUpperCase()}
-          />
-          <LabelSection
-            top="24em"
-            left="2.5em"
-            category="Time"
-            subtitle={`(${event.timeZoneAbbr})`}
-            title={event.formattedTimeSlot.toUpperCase()}
-          >
-            <StyledActionLink
-              href={event.googleCalendarUrl}
-              target="_blank"
-              rel="noreferrer"
+        <StyledArticle left>
+          <StyledEventDetailsContainer>
+            <LabelSection
+              top="10em"
+              left="2.5em"
+              category="Date"
+              title={event.formattedDate.toUpperCase()}
+            />
+            <LabelSection
+              top="24em"
+              left="2.5em"
+              category="Time"
+              subtitle={`(${event.timeZoneAbbr})`}
+              title={event.formattedTimeSlot.toUpperCase()}
             >
-              Add to Google Calendar
-            </StyledActionLink>
-          </LabelSection>
-          <LabelSection
-            top="38em"
-            left="2.5em"
-            category="Location"
-            title={event.location.name.toUpperCase()}
-          >
-            {event.location.streetAddress} <br />
-            {event.location.neighborhood}, {event.location.state}{" "}
-            {event.location.zipCode}
-            <StyledActionLink
-              margin="0.5em"
-              href={event.location.googleMapsUrl}
-              target="_blank"
-              rel="noreferrer"
+              <StyledActionLink
+                href={event.googleCalendarUrl}
+                target="_blank"
+                rel="noreferrer"
+              >
+                Add to Google Calendar
+              </StyledActionLink>
+            </LabelSection>
+            <LabelSection
+              top="38em"
```

Comment:
Are the `top`, `left`, etc. props used by `<LabelSection />`? I think the changes to this component omitted these props.